src/sage/schemes/elliptic_curves/ell_curve_isogeny.py

```diff
@@ -889,15 +889,15 @@ class EllipticCurveIsogeny(EllipticCurveHom):
     Examples over relative number fields used not to work (see :trac:`16779`)::
 
         sage: pol26 = hilbert_class_polynomial(-4*26)
-        sage: pol = NumberField(pol26,'a').optimized_representation()[0].polynomial()   # optional - sage.rings.number_field
-        sage: K.<a> = NumberField(pol)                                                  # optional - sage.rings.number_field
-        sage: j = pol26.roots(K)[0][0]                                                  # optional - sage.rings.number_field
-        sage: E = EllipticCurve(j=j)                                                    # optional - sage.rings.number_field
-        sage: L.<b> = K.extension(x^2 + 26)                                             # optional - sage.rings.number_field
-        sage: EL = E.change_ring(L)                                                     # optional - sage.rings.number_field
-        sage: iso2 = EL.isogenies_prime_degree(2); len(iso2)                            # optional - sage.rings.number_field
+        sage: pol = NumberField(pol26,'a').optimized_representation()[0].polynomial()   # optional - sage.rings.number_field, long time
```

Comment:
Also here

Reply:
Oh, thanks... I can do a grep and find all of those. Is there a style guide about this? I was often unsure which column to place the first `#`, separation between labels, etc. The only rule I know is that the first `#` needs to have 2 spaces before. Other than that, every convention I could think about is represented in some part of the code...

E.g. some places do `# optional - A   # optional - B` but other places do `# optional - A B`, etc.

I'm not even sure about the "legal" syntax, much less about the "preferred" style.

Reply:
We should definitely add a style guide for this; and a linter/fixer for these would probably also be useful (see #35401).

Both forms are correct. The style `# optional - A   # optional - B` in many places comes from using my simple editor macros.